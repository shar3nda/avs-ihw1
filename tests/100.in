10
0 4 9 0 5 5 10 10 -1 2