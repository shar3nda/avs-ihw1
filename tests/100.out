-2 9 4 -2 -1 -1 4 1